### PR TITLE
Shorten + simplify CoC

### PR DESCRIPTION
**Pull request by nathancarnes**
This is a much shorter version. My goals were to remove ambiguous language where possible, and remove redundancy. I think I was able to do that without removing any protections.

This is _very_ heavily influenced by the Contributor Covenant v1.4.0.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ We are committed to providing a friendly, safe and welcoming environment for exp
 
 ## Purpose
 
-Diversity is critical strength of our community, but it can also lead to communication issues and unhappiness. The purpose of our Community Code of Conduct is to ensure that all participants have the best possible experience while fostering an open and welcoming environment.
+The purpose of our Community Code of Conduct is to ensure that all participants have the best possible experience, while fostering an open and welcoming environment.
```
Comment (nathancarnes) [inline README.md:5]: I feel that the removed text here is easy to misinterpret.

**Comment by tantalic** [comment]
@nathancarnes - Based on discussions in Slack, I'd propose we revert commit f2733fb and then move forward with the rest of the changes in this PR. We can then open another issue/PR with the goal to have simplify that section while still having clearly outlines steps/procedures. (If you'd prefer, I can take a first pass at that this time around.)

**Comment by nathancarnes** [comment]
@tantalic I'm good with that. If you're willing to take the first crack, that'd be great!

**Comment by tantalic** [comment]
@nathancarnes will do

**Comment by tantalic** [comment]
@nathancarnes if you revert f2733fb I will merge this PR

**Comment by nathancarnes** [comment]
@tantalic doneski

**Pull request merged**